modules/launcher/src/test/java/org/mule/module/launcher/DeploymentServiceTestCase.java

```diff
@@ -3364,7 +3365,7 @@ private void deployAfterStartUp(ApplicationFileBuilder applicationFileBuilder) t
         final MuleRegistry registry = getMuleRegistry(app);
 
         // mule-app.properties from the zip archive must have loaded properly
-        assertEquals("mule-app.properties should have been loaded.", "someValue", registry.get("myCustomProp"));
+        assertThat("mule-app.properties should have been loaded.", "someValue", equalTo(registry.get("myCustomProp")));
```

Comment:
inver the order of someValue and the registry.get. In hamcrest, expeted value goes in the matcher